### ISSUE TITLE
Add anonymous toggle for damus NWC donation status [feature request]

### ISSUE THREAD

**Issue by alltheseas** [damus-io/damus#1141]
### acceptance criteria

https://damus.io/note1j66fnsyq0y9r2u4k47e32672sxkdz72mrmhj2d3c9jcc4peh0aesrvgugj

**Comment by alltheseas** [comment]
NWC, and zap donations to Damus devs must be re-examined in light of apple's 1984 zap ban

**Comment by jb55** [comment]
On Wed, Jun 28, 2023 at 11:34:52AM -0700, alltheseas wrote:
>NWC, and zap donations to Damus devs must be re-examined in light of apple's 1984 zap ban

damus nwc donations have already been removed

**Comment by alltheseas** [comment]
> On Wed, Jun 28, 2023 at 11:34:52AM -0700, alltheseas wrote:
> >NWC, and zap donations to Damus devs must be re-examined in light of apple's 1984 zap ban
> 
> damus nwc donations have already been removed
> 

Yes. Do you think it's worthwhile exploring NWC for profile zaps?

**Comment by alltheseas** [comment]
Closing as part of a backlog triage — we're narrowing focus to the most actionable issues to reduce noise for contributors.

If this is still relevant, please reopen with updated context or reproduction steps. Thanks for reporting!
